src/services/IdentityNowClient.ts

```diff
@@ -1110,6 +1110,215 @@ export class IdentityNowClient {
 			const path = `/beta/entitlements/${entitlementId}`;	
 			return await this.patchResource(path, JSON.stringify(payload));
 		}
+
+	/**
+	 * List Access Profiles
+	 * @returns list of access profiles
+	 * Added by richastral 06/07/2023
+	 */
+	public async getAccessProfiles(): Promise<any> {
```

Comment:
I have started to use the sailpoint-api-client (aka the TypeScript SDK).
So it limits the code I need to write AND the definition of data model/interfaces.
I'll take care of this